assets/stylesheets/common/discourse-post-event.scss

```diff
@@ -28,14 +27,16 @@ $interested: #fb985d;
   }
 
   .discourse-post-event-widget {
-    border: 5px solid var(--primary-low);
+    box-shadow: 0px 0px 0px 3px var(--primary-100);
+    border: 1px solid var(--primary-300);
     display: flex;
     background: var(--secondary);
     margin: 5px 0;
     flex-direction: column;
     flex: 1 0 auto;
-    max-width: 100%;
+    max-width: calc(100% - 2px - 6px);
```

Comment:
what are these magic numbers?

Reply:
The numbers are the 1px border on each side (1+1 = 2px) and the box shadow on each side (3 + 3 = 6px). This way, the total width of the box with border & box shadow always fit inside its container.